lib/urlwatch/command.py

```diff
@@ -212,6 +212,31 @@ def modify_urls(self):
             print('Adding %r' % (job,))
             self.urlwatcher.jobs.append(job)
 
+        if self.urlwatch_config.change_location is not None:
+            new_loc = self.urlwatch_config.change_location[1]
+            # Ensure the user isn't overwriting an existing job with the change.
+            if new_loc in (j.get_location() for j in self.urlwatcher.jobs):
+                print(f'The new location "{new_loc}" already exists for a job. Use that '
+                      'existing job or choose a different value.')
```

Comment:
Style nitpick:
```
                print(f'The new location "{new_loc}" already exists for a job. '
                      'Use that existing job or choose a different value.')
```

Semantic feedback:
What does "Use that existing job" mean in this context? Apart from accidental use (in which case the user sees the error message and goes on to correct their mistake), if people WANT to merge two different histories, maybe we should somehow allow this (and choose one or the other in the job list, maybe the "target" job gets overwritten using `--overwrite` or `--force` or something?). I guess the workaround now would be to just delete the target job from the jobs list manually (keeping historic entries in the database) and then do the rename? If this is good enough, then I suggest a wording change:

```
                print(f'The new location "{new_loc}" already exists for a job. '
                      'Delete the existing job or choose a different value.')
```

Reply:
I was thinking that if somebody was updating a URL to watch example.com but they already have a job that watches example.com, then they can just use the existing example.com job to get whatever data they need.  Hence "Use", though I like "Delete" better.